### PR TITLE
br: remove backup and restore principle

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Remove unnecessary BR principles.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### CODE REVIEW

**Comment by TomShawn** [comment]
@overvenus Please involve a technical review.

**Comment by 3pointer** [comment]
LGTM, but should we move these BR principles to new file?

**Comment by overvenus** [comment]
> but should we move these BR principles to new file?

I don't think so, because there is a doc about the principles in pingcap/br repo already.

**Comment by overvenus** [comment]
PTAL @TomShawn

**Comment by TomShawn** [comment]
> > but should we move these BR principles to new file?
> 
> I don't think so, because there is a doc about the principles in pingcap/br repo already.

What about referencing to that doc in this doc?

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -28,52 +28,16 @@ Backup & Restore（以下简称 BR）是 TiDB 分布式备份恢复的命令行
 
 BR 是分布式备份恢复的工具，它将备份或恢复操作命令下发到各个 TiKV 节点。TiKV 收到命令后执行相应的备份或恢复操作。在一次备份或恢复中，各个 TiKV 节点都会有一个对应的备份路径，TiKV 备份时产生的备份文件将会保存在该路径下，恢复时也会从该路径读取相应的备份文件。
```
Comment (overvenus) [inline br/backup-and-restore-tool.md:29]:
```suggestion
BR 是分布式备份恢复的工具，它将备份或恢复操作命令下发到各个 TiKV 节点。TiKV 收到命令后执行相应的备份或恢复操作。在一次备份或恢复中，各个 TiKV 节点都会有一个对应的备份路径，TiKV 备份时产生的备份文件将会保存在该路径下，恢复时也会从该路径读取相应的备份文件。更多信息请参阅[备份恢复设计方案](https://github.com/pingcap/br/blob/980627aa90e5d6f0349b423127e0221b4fa09ba0/docs/cn/2019-08-05-new-design-of-backup-restore.md)
```

**Pull request merged**